Automattic/jetpack
# AI Assistant: Release Breve for 50% of sites

Fixes #

## Proposed changes:

* Adds other digits to the release check

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On a simple site with blog id ending in 1, 3, 4, 5 or 8, check that Breve is not enabled
* On a simple site with blog id ending in 0, 2, 6, 7 or 9, check that Breve is enabled
